pkgs/development/ocaml-modules/functoria/default.nix

```
@@ -4,13 +4,14 @@
 
 buildDunePackage rec {
   pname   = "functoria";
-  version = "4.2.0";
+  version = "4.3.3";
```

```suggestion
  version = "4.3.4";
```
Just had a new release.